.gitignore

```diff
@@ -9,6 +9,7 @@ npm-debug.log
 
 # other
 .next
+/examples/**/yarn.lock
```

Comment:
It would be great to leave a short explanations of why you don't want `yarn.lock` file as a `.gitignore`, as I guess future readers might get slightly confuse by that.